packages/components/src/checkbox-control/README.md

```diff
@@ -1,42 +1,88 @@
 # CheckboxControl
 
-CheckboxControl component is used to generate a checkbox input field.
+Checkboxes allow the user to select one or more items from a set.
 
+![](https://make.wordpress.org/design/files/2019/02/CheckboxControl.png)
 
-## Usage
+Selected and unselected checkboxes
+
+## Table of contents
+
+1. [Design guidelines](http://#design-guidelines)
+2. [Development guidelines](http://#development-guidelines)
+3. [Related components](http://#related-components)
+
+## Design guidelines
+
+### Usage
+
+#### When to use checkboxes
+
+Use checkboxes when you want users to:
+
+- Select one or multiple items from a list.
+- Open a list containing sub-selections.
+
+![](https://make.wordpress.org/design/files/2019/02/select-from-list.png)
+
+**Do**
+Use checkboxes when users can select multiple items from a list. They let users select more than one item.
+
+![](https://make.wordpress.org/design/files/2019/02/many-form-toggles.png)
+
+**Don’t**
+Don’t use toggles when a list consists of multiple options. Use checkboxes — they take up less space.
+
+![](https://make.wordpress.org/design/files/2019/02/checkbox-sublist.gif)
+
+Checkboxes can be used to open a list containing sub-selections.
+
+#### Parent and child checkboxes
+
+Checkboxes can have a parent-child relationship, with secondary options nested under primary options.
+
+![](https://make.wordpress.org/design/files/2019/02/checkbox-parent.gif)
+
+When the parent checkbox is *checked*, all the child checkboxes are checked. When a parent checkbox is *unchecked*, all the child checkboxes are unchecked.
+
+![](https://make.wordpress.org/design/files/2019/02/mixed-checkbox.png)
+
+If only a few child checkboxes are checked, the parent checkbox becomes a mixed checkbox.
+
+## Development guidelines
+
+### Usage
 
 Render an is author checkbox:
 ```jsx
 import { CheckboxControl } from '@wordpress/components';
 import { withState } from '@wordpress/compose';
 
 const MyCheckboxControl = withState( {
-	isChecked: true,
```

Comment:
We lose some useful/standards-compliant tabbing here.

Reply:
I believe this is now fixed. Do you mind checking? If

Reply:
> I believe this is now fixed. Do you mind checking? If

Looks fine now. Thanks 👍